packages/agent/src/server.ts

```diff
@@ -78,3 +98,17 @@ function listen(app: xp.Express, port?: number): Operation {
     })
   };
 };
+
+function buildDriver({ browser, headless }: LaunchOptions): Promise<WebDriver> {
+  let builder = new Builder().forBrowser(browser);
+  switch (browser) {
+    case 'chrome':
+      let chromeOptions = new ChromeOptions();
+      chromeOptions = headless ? chromeOptions.headless() : chromeOptions;
+      return builder.setChromeOptions(chromeOptions).build();
+    case 'firefox':
+      let firefoxOptions = new FirefoxOptions();
+      firefoxOptions = headless ? firefoxOptions.headless() : firefoxOptions;
+      return builder.setFirefoxOptions(firefoxOptions).build();
```

Comment:
I think it's not necessary to set these options conditionally. If a different browser is used the additional options are ignored.